README.md

```diff
@@ -58,3 +138,37 @@ GeoServer will then be available at:
 ```
 http://localhost:9090
 ```
+
+## Running using Tomcat and IntelliJ
+
+Install Tomcat 8.5
+
+Install Java version 11
+
+Set the maven profile to `tomcat`.
+
+![img.png](img.png)
+
+Create a `src/main/webapp/META-INF/context.xml` file using the supplied `src/main/webapp/META-INF/context-sample.xml` as 
+a template and update to use the required database datastore.
+
+Create and populate a `src/main/webapp/data` directory with custom content.
```

Comment:
what is `custom content` you mentioned in this statement, and where can I find them? Thanks @craigrose